api/api/api.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
```

Comment:
Let's move this up one directory. Importing `github.com/talos-systems/talos/api/api` can be cleaned up to `github.com/talos-systems/talos/api`

Reply:
bump